hack/perfscale-tests.sh

```diff
@@ -63,22 +71,24 @@ if [[ ${PERFAUDIT} == "true" || ${PERFAUDIT} == "True" ]]; then
     export PERFSCALE_WORKLOAD=$_perfscale_workload_warmup
     # run small test to warm up the system and be able to collect metrics
     perftest ${additional_test_args} ${FUNC_TEST_ARGS}
-    # wait 30 before running the test to let prometheus scrape metrics
-    echo "Sleeping 30 to let prometheus scrape all metrics"
-    sleep 30
     export PERFSCALE_WORKLOAD=$_perfscale_workload_tmp
 fi
 
-start_timestamp=$(date -u +%Y-%m-%dT%TZ)
+start_timestamp=$(get_timestamp)
+# wait 30 before running the test to let prometheus scrape metrics
+echo "Sleeping 30 to let prometheus scrape all metrics"
+sleep 30
 # run the test
-perftest ${additional_test_args} ${FUNC_TEST_ARGS}
+time perftest ${additional_test_args} ${FUNC_TEST_ARGS}
 
 # run audit tool to dump metrics
 if [[ ${PERFAUDIT} == "true" || ${PERFAUDIT} == "True" ]]; then
-    # wait 180s after finished the test. More info in https://github.com/kubevirt/kubevirt/issues/7083
-    stop=$(date -u +%Y-%m-%dT%TZ)
-    echo "Sleeping 180s to let prometheus scrape all metrics"
-    sleep 180s
-    stop_timestamp=$(date -u +%Y-%m-%dT%TZ)
+    # wait 30s after finished the test. More info in https://github.com/kubevirt/kubevirt/issues/7083
+    echo "Sleeping 30s to let prometheus scrape all metrics"
+    sleep 30s
+    stop_timestamp=$(get_timestamp)
     perfaudit $start_timestamp $stop_timestamp
 fi
+
+echo "start_timestamp= $start_timestamp"
+echo "stop_timestamp= $stop_timestamp"
```

Comment:
I don't see `stop_timestamp` being set besides the above `if`? Does it matter?

Reply:
Will fix it. Thanks